src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -116,3 +116,24 @@
                 - name: Remove fourth NIC
                   text: az <command> -n name -g group --remove networkProfile.networkInterfaces 3
             """.format(generic_update_help)
+
+helps['vmss get-instance-view'] = """
+    type: command
+    parameters: 
+        - name: --ids
+          short-summary: "One or more scale set or specific VM instance IDs. If provided, no other 'Resource Id' arguments should be specified."
```

Comment:
It can...and I started to, but I didn't want the help files to become crazy patchworks like the params file. I figured you wouldn't either so I split them out. :)

---
In reply to: [77388591](https://github.com/Azure/azure-cli/pull/803#discussion_r77388591) [](ancestors = 77388591)